src/components/ScholarshipAmountField.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  TextField,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import AmountType from '../types/AmountType';
+
+const useStyles = makeStyles({
+  formControl: {
+    minWidth: 200,
+  },
+});
+
+function ScholarshipAmountField(props) {
+  const classes = useStyles();
+  const {
+    amountType,
+    minAmount,
+    maxAmount,
+    onTypeChange,
+    onMinChange,
+    onMaxChange,
+    updateMinMax,
+  } = props;
+
+  function displayAmountFields() {
+    if (amountType === AmountType.Fixed) {
+      return (
+        <TextField
+          label="Amount"
+          type="number"
+          InputLabelProps={{ shrink: true }}
+          fullWidth
```

Comment:
I suggest not making these full width for visual consistency. Imagine you have lots of form fields below and then you toggle this. All of the sudden everything below shifts up/down. Take a look at https://images.app.goo.gl/c4vCztKPDjp8sz5u5 to see what I mean